core/src/banking_stage.rs

```diff
@@ -443,46 +443,46 @@ impl BankingStage {
         // Many banks that process transactions in parallel.
         let bank_thread_hdls: Vec<JoinHandle<()>> = (0..num_threads)
             .map(|i| {
-                let (verified_receiver, unprocessed_transaction_storage) =
+                let (packet_receiver, unprocessed_transaction_storage) =
                     match (i, should_split_voting_threads) {
                         (0, false) => (
```

Comment:
it's so tempting to reorder this match arms. but, I'm abiding to no functional change. (otherwise, metrics meaning would be changed).